stage0/run.go

```diff
@@ -145,6 +144,27 @@ func imageNameToAppName(name types.ACIdentifier) (*types.ACName, error) {
 	return types.MustACName(sn), nil
 }
 
+// deduplicateMPs removes mounts with duplicated paths. If there's more than
+// one mount with the same path, it keeps the first one encountered.
```

Comment:
Actually, I'm refering to the schema.Mount structure so it is correct. I'll upper-case "mount"